### PR TITLE
add contributing guide

### PR DESCRIPTION
**Pull request by Tushkiz**

### CODE REVIEW

**Comment by Tushkiz** [comment]
@CompuIves Please review 😬

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+> This will add the original repository as a "remote" called "upstream,"
+> Then fetch the git information from that remote, then set your local `master`
+> branch to use the upstream master branch whenever you run `git pull`.
+> Then you can make all of your pull request branches based on this `master`
+> branch. Whenever you want to update your version of `master`, do a regular
+> `git pull`.
+
+## Submitting a Pull Request
+
+Please go through existing pull requests and check if somebody else is already working on this.
```
Comment (CompuIves) [inline CONTRIBUTING.md]: Maybe also mention to look through the issues, we have a tag for it.

**Comment by CompuIves** [comment]
Yas! This is great! Left one comment and I have another request. Maybe it would be good to create a PR template where we mention that you should add yourself to the contributors list?

File: CONTRIBUTING.md (hunk 1 above)
Comment (Tushkiz) [reply CONTRIBUTING.md]: ohh yes, the `someone working on it` tag

**Comment by CompuIves** [comment]
Awesome! Thanks!

**Comment by Tushkiz** [comment]
We can close #221 now :bowtie:

File: CONTRIBUTING.md (hunk 1 above)
Comment (zixuan75) [reply CONTRIBUTING.md]: Someone working on it?

**Pull request merged**